users/admin.py

```diff
@@ -20,7 +20,7 @@ class CustomUserAdmin(UserAdmin):
     )
     add_fieldsets = ((None, {"classes": ("wide"), "fields": ("email", "password1", "password2")}),)
     form = CustomUserChangeForm
-    add_form = CustomUserCreationForm
+    add_form = UserCreationForm
```

Comment:
When you're doing changes like this please always add small description to commit because it's hard now to know what you wanted to do here.

Reply:
Done. Please check that this description is sufficient.